examples/rgat_conv.py

```diff
@@ -0,0 +1,88 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from rgat_conv import RGATConv
+from torch.nn import Linear
+from torch.nn.utils import clip_grad_value_
+
+from torch_geometric.datasets import Entities
+
+
+class Net(nn.Module):
+    def __init__(self, d=2):
+        super(Net, self).__init__()
+
+        self.nclass = 4
+        self.readout = "add"
+        self.in_channels = 16
+        self.out_channels = 25
+        self.d = d
+
+        self.conv_layers = nn.ModuleList([
+            RGATConv(16, 20, num_relations=90, num_bases=35, mod="additive",
+                     attention_mechanism="within-relation",
+                     attention_mode="multiplicative-self-attention", heads=2,
+                     d=2),
+            RGATConv(80, 25, num_relations=90, num_blocks=2, mod=None,
+                     attention_mechanism="across-relation",
+                     attention_mode="additive-self-attention", heads=2, d=1,
+                     dropout=0.6, edge_dim=16, bias=False),
+        ])
+
+        self.fc = Linear(16, self.in_channels, bias=False)
```

Comment:
The naming is a bit off. Shall we name this `self.lin1` and `self.lin2`?
```suggestion
        self.lin1 = Linear(16, self.in_channels, bias=False)
```